pyblish_lite/window.py

```diff
@@ -845,10 +963,34 @@ def on_was_processed(self, result):
                     plugins_filter = self.data["models"]["filter"]
                     plugins_filter.add_inclusion(role="families", value=f)
 
+        error = result.get('error')
+        if error:
+            fname, line_no, func, exc = error.traceback
+            records = result.get('records') or []
```

Comment:
I tried merging the currently conflicting files, but discovered that "records" are currently a mixture of LogRecords and dictionary items. It just so happens the conflicting files depend on those being instances of LogRecord, which is what Pyblish Base is passing to Lite via the `results` dictionary.

This appears to be the offending line. It's looking for `records`, which is will find, as these are the LogRecord instances passed from Base. But then it's appending dictionary items to this list, which I'm not sure is intended.

Having a look at this now.. but let me know if any of this rings any bell.